### PR TITLE
Add Syncthing add-on

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

**Comment by awiouy** [comment]
I trust everything is in order now. Sorry for useless pull request 105.

Incidentally, it would be cool to make this addon available to OpenELEC 5.0 and 6.0, since the synthing binary also runs on them.

Regards,
@

File: addons/service/system/syncthing/package.mk [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+if [ "$TARGET_ARCH" = "arm" ]; then
```
Comment (lrusak) [inline addons/service/system/syncthing/package.mk]: Please move this if block to below ```PKG_AUTORECONF="no"```

**Comment by lrusak** [comment]
Also change the commit message to something more descriptive
```syncthing: add initial add-on```

**Comment by awiouy** [comment]
Done, thanks, regards, @

**Comment by lrusak** [comment]
Just realized that you are using the precompiled binaries, this is not what we want. Please change it to use the source code so that it can be compiled for each specific project.

**Comment by awiouy** [comment]
Great!
I will advertize and document this on the OpenELEC and Syncthing fora as soon as this is committed.
@

**Comment by awiouy** [comment]
Hello!
Is there a problem with this pull request? Anything I should fix?
@

**Comment by chewitt** [comment]
What "projects" has this been tested to run on?

**Comment by awiouy** [comment]
This has been tested to run on RPi and RPi2 (same binary)

**Comment by lrusak** [comment]
Thanks for your contribution :+1:

**Pull request merged**